mteb/abstasks/TaskMetadata.py

```diff
@@ -144,3 +146,6 @@ class TaskMetadata(BaseModel):
 
     text_creation: TEXT_CREATION_METHOD | None
     bibtex_citation: str | None
+
+    n_samples: dict[str, int]
```

Comment:
Maybe we should specify these as dict[SPLIT_TYPE, int]? This can be something as simple as a type-alias, but makes it semantically clearer.

Reply:
Might also be nice to provide a little utility that gets this info, and document the utility here.

Reply:
it would be possible to check it using the eval_splits attr.